scripts/push-documentation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+setup_git() {
+  git config --global user.email "nakul.korde@intel.com"
+  git config --global user.name "Nakul Korde"
+}
+
+commit_website_files() {
+  mkdir upload_docs
+  cd upload_docs
+  git clone https://github.com/OPAE/opae.github.io.git
+  cd doc-testing
+  cp -r ../../mybuild_docs/sphinx/html/* .
+  git add -A
+  git commit --message "Travis build: Update Documentation with traceback: second test commit $TRAVIS_BUILD_NUMBER"
```

Comment:
traceback in some languages refers to a stack trace. This message might confuse people (like me).

Reply:
Thanks for pointing out; rectified in new commit.

Reply:
Will you change the commit message later (this looks like a test)?